Steer/src/O2MCApplication.cxx

```diff
@@ -114,6 +115,35 @@ void O2MCApplicationBase::InitGeometry()
   }
 }
 
+bool O2MCApplicationBase::MisalignGeometry()
+{
+  for (auto det : listActiveDetectors) {
+    if (dynamic_cast<o2::base::Detector*>(det)) {
+      ((o2::base::Detector*)det)->addAlignableVolumes();
+    }
+  }
+
+  auto b = FairMCApplication::MisalignGeometry();
+  // we use this moment to stream our geometry (before other
+  // VMC engine dependent modifications are done)
+
+  std::stringstream geomss;
+  geomss << "O2geometry";
+  auto pid = getpid();
```

Comment:
As we do not use the pid in the names of the other output files, eg. , I would suggest not to do it here as well.  If this feature is needed, it should be better implemented in a more generic way for all files.

Reply:
Can take it out. It was needed in the previous version of export from which I just copied the code.